scripts/configs/example.yaml

```diff
@@ -3,15 +3,15 @@
 APPROACHES:
   my-oracle:  # used in constructing the experiment ID
     NAME: "oracle"
-    FLAGS: []
+    FLAGS: {}
```

Comment:
it evidently didn't matter for the existing code, but i realized that the right yaml syntax for an empty dictionary is this